README.md

```diff
@@ -4,14 +4,20 @@
 
 > In this project, I will create a personal portfolio. A portfolio website is a key job searching tool for a web developer. In this project, I will precisely replicate one of the *portfolio templates* from Microverse Figma. 
 
-![screenshot](./images/app_screenshot.png)
+![screenshot](./images/Banner.png)
+![screenshot](./images/Contact.png)
 
 Additional description about the project and its features.
 
 ## Built With
 
 - Html/Css
 
+## Additional Tools
+
+- Figma
+- stylelint
+
 
 ## Authors
 
```

Comment:
**REQUIRED:** Please kindly personalize the MIT.md file, add your name, project name and delete unnecessary lines, so it will look more professional.

![image](https://user-images.githubusercontent.com/77872355/129119024-1fca7405-c38e-4483-81ad-532face61607.png)